controllers/main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/dchest/captcha"
```

Comment:
This import should be with the other github imports, not with the std lib imports.